src/main/scala/za/co/absa/hyperdrive/trigger/scheduler/utilities/Configs.scala

```diff
@@ -83,6 +83,8 @@ object SchedulerConfig {
 object ExecutorsConfig {
   def getThreadPoolSize: Int =
     Configs.conf.getInt("scheduler.executors.thread.pool.size")
+  def getExecutablesFolder: String =
```

Comment:
I think we should use `val`s instead of `def`s in the `Configs.scala`, but especially for this property. If it's not set in the `application.properties`, the shell job will fail silently, because it wraps everything in a `Try`, so the exception here will be swallowed.
In theory, we lose the option to change the property at runtime, but I think we shouldn't do that anyway

Reply:
Ok